.github/workflows/build.yml

```diff
@@ -22,28 +22,24 @@ jobs:
         uses: cisagov/setup-env-github-action@develop
       - uses: actions/checkout@v3
       - id: setup-python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
```

Comment:
Please update the other uses of `actions/setup-python`:
https://github.com/cisagov/skeleton-packer/blob/f5f8bb83bdb0cd9a13c8e6a71a75a302b1520315/.github/workflows/build.yml#L111
https://github.com/cisagov/skeleton-packer/blob/f5f8bb83bdb0cd9a13c8e6a71a75a302b1520315/.github/workflows/build.yml#L164